servicecontrol/troubleshooting.md

```diff
@@ -195,6 +203,8 @@ If many indexes are affected it may be easier to rebuild all indexes, although t
 - Delete the `Indexes` folder
 - Start the ServiceControl instance
 
+Info: Sometimes deleting the 'Indexes' folder or the database folder can cause the audit instance to not start. If that comtinues to happen, try running the 'ServiceControl.Audit.exe /setup' in powershell
```

Comment:
Really?

Reply:
@WilliamBZA - updated text to be more explicit